lib/open_mic.rb

```diff
@@ -0,0 +1,18 @@
+class OpenMic
+
+  attr_reader :location, :date, :performers
+  def initialize(info_hash)
+    @location = info_hash[:location]
+    @date = info_hash[:date]
+    @performers = []
```

Comment:
perfect!! _great_ work identifying this as a hash and using square brackets to access the appropriate values